apps/dashboard/app/models/user_configuration.rb

```diff
@@ -56,8 +56,9 @@ class UserConfiguration
     ConfigurationProperty.property(name: :dashboard_title, default_value: 'Open OnDemand', read_from_env: true),
   ].freeze
 
-  def initialize
+  def initialize(request_hostname: nil)
```

Comment:
Because of the default value to `nil`, I am assuming that the `request_hostname` is optional.

If that is the case, I will only add `request_hostname` to the creation of UserConfiguration in the tests to check the profile selection logic.

Reply:
Yea there's not going to be a good default, so just handling `nil` in layers below is fine.